blockbin/src/components/InputForm.js

```diff
@@ -1,16 +1,73 @@
 import React, { Component } from 'react';
+import Web3 from 'web3';
 
 import '../App.css';
 
+const CONTRACT_ADDRESS = '0xa022cc6ff8895fd07b03c480d46f20c412716fdc';
```

Comment:
This seems like something that should be put in configs? I'm pretty sure that we'll all get different addresses on our test networks, and once we deploy this "for real" we'll have one "prod" contract that's again different.